File: Project/app/src/main/java/com/example/android/project/MainActivity.java
package com.example.android.project;

import android.arch.persistence.room.Room;
import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {
    public static Database database;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        database= Room.databaseBuilder(getApplicationContext(),Database.class,"userdb").allowMainThreadQueries().build();

        final Button write = (Button) findViewById(R.id.writeButton);

        write.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent writeIntent = new Intent(MainActivity.this,write.class);
                startActivity(writeIntent);
            }
        });


        final Button read = (Button) findViewById(R.id.readButton);

        read.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent readIntent = new Intent(MainActivity.this,read.class);
                startActivity(readIntent);
            }
        });

        final Button update = (Button) findViewById(R.id.updateButton);

        update.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent updateIntent = new Intent(MainActivity.this,update.class);
                startActivity(updateIntent);
            }
        });
    }
}

File: LossLessLife00/app/src/main/java/com/example/android/losslesslife00/MainActivity.kt
package com.example.android.losslesslife00

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val signUpButton = findViewById<Button>(R.id.signUp) as Button
        val signInButton = findViewById<Button>(R.id.signIn) as Button

        signUpButton.setOnClickListener{
            val intent = Intent(this,SignUp::class.java)
            startActivity(intent)
        }

        signInButton.setOnClickListener{
            val intent = Intent(this,LogIn::class.java)
            startActivity(intent)
        }

    }
}
